Sources/Components/HorizontalSlide/HorizontalSlideTransition.swift

```diff
@@ -5,10 +5,32 @@ import UIKit
 }
 
 public class HorizontalSlideTransition: NSObject, UIViewControllerTransitioningDelegate {
+
     @objc public weak var delegate: HorizontalSlideTransitionDelegate?
 
+    private let containerPercentage: () -> CGFloat
```

Comment:
I think a would have defined a struct instead. Something like:

```swift
struct ContainerSize {
    let portrait: CGFloat
    let landscape: CGFloat?

    static var `default`: ContainerSize {
        if iPad {
            return ContainerSize(portrait: 0.4, landscape: 0.3)
        } else {
            return ContainerSize(portrait: 0.6)
        }
}
```
And then with an init
`public init(width size: ContainerSize)`

Reply:
Didn't even think of this as a possibility! This looks way better 👍